helm-chart-sources/pulsar/templates/openid/kafka-configmap.yaml

```diff
@@ -0,0 +1,34 @@
+#
+#  Copyright 2022 DataStax, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#
+
+{{- if and .Values.openid.enabled .Values.openid.withS4k }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "openid-s4k-config"
```

Comment:
I think we try to add the cluster name in the config map name, but I might be misremembering. Will you double check how some of the other config map names are created?